backend/src/schema/resolvers/follow.spec.js

```diff
@@ -41,44 +70,57 @@ describe('follow', () => {
   describe('follow user', () => {
     describe('unauthenticated follow', () => {
       it('throws authorization error', async () => {
-        const client = new GraphQLClient(host)
-        await expect(client.request(mutationFollowUser('u2'))).rejects.toThrow('Not Authorised')
+        authenticatedUser = null
+        const { errors } = await mutate({
+          mutation: mutationFollowUser('u2'),
+        })
+        expect(errors[0]).toHaveProperty('message', 'Not Authorised!')
       })
     })
 
     it('I can follow another user', async () => {
-      const res = await clientUser1.request(mutationFollowUser('u2'))
-      const expected = {
-        follow: true,
-      }
-      expect(res).toMatchObject(expected)
+      const { data: result } = await mutate({
+        mutation: mutationFollowUser(user2.id),
+      })
+
+      const expected = { follow: true }
```

Comment:
```suggestion
      const expected = { name: 'Name of the followed user', followedBy: [ { name: 'Me' } ], followedByCurrentUser: true }
```

Reply:
@roschaefer good suggestion. Definitely will use it with further steps :)